client/src/components/Pledge.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import axios from 'axios';
+import ProjectOwner from './ProjectOwner.jsx';
+import PledgeOption from './PledgeOption.jsx';
+
+class Pledge extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      owner: {
+        name: 'Name',
+        iconUrl: 'https://images.pexels.com/photos/2872767/pexels-photo-2872767.jpeg?auto=compress&cs=tinysrgb&h=750&w=1260',
+        created: 0,
+        backed: 0,
+        aboutMe: 'About Me'
+      },
+      options: []
+    }
+    this.fetchOwner = this.fetchOwner.bind(this);
+    this.fetchOptions = this.fetchOptions.bind(this);
+  }
+
+  componentDidMount() {
```

Comment:
I think we are supposed to move these invocations into the constructor and not keep them in componentDidMount. Leslie suggested that during one of the Q&A's.

Reply:
Good tip, I'll have to check that out!